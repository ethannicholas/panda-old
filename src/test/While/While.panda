var total := 1
while total < 10000
    total *= 2
Console.writeLine(total)

while true {
    total := 0
    break
}
    
Console.writeLine(total)

while "Hello" = "Goodbye"
    total := 128
    
Console.writeLine(total)
