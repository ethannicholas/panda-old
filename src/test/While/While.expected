16384
0
0
